playbooks/roles/document_hosting_k8s/tasks/main.yaml

```diff
@@ -290,10 +290,23 @@
         ingressClassName: "{{ document_hosting.ingress_class | default(omit) }}"
         tls:
           - hosts:
-              - "{{ document_hosting.server_name }}"
+              - "{{ document_hosting.server_name.0 }}"
+            secretName: "{{ document_hosting_ssl_cert_name }}"
+          - hosts:
+              - "{{ document_hosting.server_name.1 }}"
             secretName: "{{ document_hosting_ssl_cert_name }}"
         rules:
-          - host: "{{ document_hosting.server_name }}"
+          - host: "{{ document_hosting.server_name.0 }}"
+            http:
+              paths:
+                - path: "/"
+                  pathType: "Prefix"
+                  backend:
+                    service:
+                      name: "{{ document_hosting_svc_name }}"
+                      port:
+                        number: 8080
+          - host: "{{ document_hosting.server_name.1 }}"
```

Comment:
this is a definitely wrong way to do this. Instead I suggest to convert ingress to ansible template and iterate using jinja (something like https://github.com/opentelekomcloud-infra/system-config/blob/main/playbooks/roles/grafana/templates/grafana-deployment.yaml.j2#L101) This way role should be capable of dealing with arbitrary count of hostnames.